docs/source/netutils/index.rst

```diff
@@ -0,0 +1,16 @@
+******************
+Netutils Functions
+******************
+
+.. toctree::
+   :maxdepth: 2
+
+   asn/index
```

Comment:
Where is config?

Reply:
The config for `asn`? It lives in its own folder under `source/netutils/asn`.

Reply:
like should there be an `config/index`?

Reply:
Totally missed those. I added a general `configs` tab. We could break it up how the code is with each file having its own tab under config. Let me know what you think and I can change it if need be.